lib/travis/client/session.rb

```diff
@@ -17,7 +17,7 @@ module Travis
   module Client
     class Session
       PRIMITIVE   = [nil, false, true]
-      SSL_OPTIONS = { :ca_file => Tools::Assets['cacert.pem'] }
```

Comment:
Thanks! Moving to system CAs is certainly something worth considering in general, but it will be easier to merge this PR if we defer that decision.